src/Sarif/Notes.cs

```diff
@@ -2,6 +2,8 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 
+using System;
```

Comment:
Formatting convention: Remove the extra blank line under the copyright notice, leaving only one (we avoid multiple consecutive blank lines).